package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "To do list is a basic website, On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.",
+  "main": "src/index.js",
```

Comment:
- [ ] To properly setup webpack you had to delete this line and replace with `private: true`.
<img width="727" alt="Screenshot 2022-03-22 at 21 29 24" src="https://user-images.githubusercontent.com/77038610/159540393-0f09323f-56a2-4a80-9f0a-138f5a648d1c.png">

Reply:
Fixed